src/Raven.Server/Utils/ShardHelper.cs

```diff
@@ -44,7 +44,7 @@ public static int GetBucketFor(ShardingConfiguration configuration, ReadOnlySpan
         {
             var bucket = GetBucketFor(lowerId);
 
-            if (configuration.Prefixed != null)
+            if (configuration?.Prefixed != null)
```

Comment:
if `configuration` is `null` something is very off... when can this happened?

Reply:
I noticed that we're passing `database?.ShardingConfiguration` [here](https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Server/Documents/Sharding/ShardedDocumentsStorage.cs#L191)
shouldn't happen, but just to be safe